Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:2.7
+
+WORKDIR /usr/src/app
+RUN echo "PATH=$PWD/local/bin:\$PATH" >> ~/.bashrc
+COPY . .
+RUN pip install --no-cache-dir -r requirements.txt
```

```suggestion
RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -yy libsasl2-dev libldap2-dev
RUN pip install --no-cache-dir -r requirements.txt
```
Amb el suport per LDAP cal tenir instal·lades algunes llibreries addicionals.